.github/integration/scripts/make_sda_credentials.sh

```diff
@@ -1,28 +1,34 @@
 #!/bin/sh
 set -e
 
+URI=http://rabbitmq:15672
+if [ -n "$PGSSLCERT" ]; then
```

Comment:
?

Reply:
Postgres requires a client certificate, so it's the simplest way of detecting if TLS is enabled or not.

Reply:
Fine enough, but could we get a comment highlighting that postgres tls usage is taken as proxy for tls usage for others?